TestServer/server.mjs

```diff
@@ -18,6 +18,44 @@ app.get('/goto/timeout/:sec', (req, res) => {
     }, ms);
 });
 
+// Simulate latency not on the html, but on the assets.
+// This way domcontentloaded is fast, but the load event is slowed down.
+
+// Dynamic route to set latency and serve the requested file
+app.get('/load_latency/:sec/index.html', (req, res) => {
```

Comment:
Consider using `ms` instead of `sec` for fine-grained control in tests. And ms is kind of "standard".

Reply:
Good point.